### DISCUSSION TITLE
batch size in training parta2 on kitti

### DISCUSSION THREAD

**Discussion by gynaa** [open-mmlab/mmdetection3d#2331] [Q&A] [answered]
Hello :)

where can I adjust the batch size? I figured it would be set in one of the configuration files, but I couldn't find it anywhere. 

Thanks!!

**Comment by JingweiZhang12** [comment, marked as answer]
@gynaa https://github.com/open-mmlab/mmdetection3d/blob/master/configs/_base_/datasets/kitti-3d-3class.py#L113
